lib/modulesync.rb

```diff
@@ -82,7 +78,7 @@ def self.run(args)
         if options[:noop]
           Git.update_noop(puppet_module, options[:branch])
         else
-          Git.update(puppet_module, files_to_manage, options[:message], options[:branch], options[:bump], options[:changelog], options[:tag], options[:tag_pattern])
+          Git.update(puppet_module, files_to_manage, options[:message], options[:branch], options[:branch_prefix], options[:bump], options[:changelog], options[:tag], options[:tag_pattern], options[:pre_commit_script], options[:amend], options[:force])
```

Comment:
In an earlier comment you asked about passing the whole options hash instead of each value...I +1 that idea, this is starting to look silly. :)